documentation/developer-guide/modules/tooling-guide/pages/java-aspect-tooling.adoc

```diff
@@ -439,3 +439,82 @@ characteristicDefinitionsUrl.ifPresent( url -> {
+
+| bamm:name   | aas:Submodel.idShort  |
+
+| bamm:preferredName   | aas:Submodel.displayName  |
+
+| bamm:description   | aas:Submodel.description  |
+
+| bamm:property   | see **bamm:Property**  |
+
+| bamm:operation   | see **bamm:Operation**   |
+
+| bamm:Aspect.urn  | aas:Submodel.semanticId  |
+
+| **Property**  | aas:Property <br> aas:SubmodelElementCollection   | The AAS type is derived from the type of the BAMM Characteristic specifying the BAMM property. Depending on the type it is decided what the resulting AAS element will be. In case of an Entity it will result in a SubmodelElementCollection. It will also be a SubmodelElementCollection if the BAMM Characteristic is of a Collection type (see the https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/characteristics.html[Characteristics taxonomy]). In all other cases an aas:Property will be generated
+
+| bamm:Property.name  | aas:Property.idShort |
+
+| bamm:Property.urn  | aas:ConceptDescription.identification.id <br> aas:Property.semanticId|
+
+| bamm:Property.preferredName   | aas:Property.displayName  |
+
+| bamm:Property.description   | aas:Property.description  |  Note: Also mapped to aas:DataSpecificationIEC61360.definition of the aas:ConceptDescription of this property
+
+| bamm:Property.exampleValue  |aas:Property.value  |
+
+| bamm:Characteristic.dataType  | aas:Property.valueType |
+
+| **Operation** |  Operation  | in/out parameters are not used in BAMM so the mapping only generates input variables and output variables in AAS
+
+| **Characteristic**  | aas:SubmodelElement, aas:ConceptDescription  | Characteristics in BAMM define the semantics of a property, which includes there types as well as links to further definitions (standards, dictionaries, etc), a natural language description and name in different languages. Type and description are separated in AAS, which is why there is not a one-to-one mapping of a Characteristic to one element in AAS but rather Characteristics are used in the mapping of Properties, first, to guide the generation process and, second, to capture semantics in ConceptDescriptions of properties with data specification "DataSpecificationIEC61360" of the AAS.
+
+| **Collection**  | aas:SubmodelElementCollection, aas:ConceptDescription  | The general remarks to Characteristics apply also for Collection type Characteristics. However, properties referencing Collections are mapped to SubmodelElementCollections. Specific properties of collections are mapped. bamm:Set is unique, bamm:SortedSet is unique and sorted, bamm:List is sorted.
+
+| **Quantifiable**  | aas:SubmodelElement, aas:ConceptDescription | The general remarks to Characteristics apply also for Quantifiable type Characteristics. Quantifiables (also Duration and Measurement) reference a unit, which is added to the ConceptDescription corresponding the mapped Characteristic.
```

Comment:
Please add `xrefs` for all elements where it makes sense (i.e., at least all Characteristics). For example, instead of `**Quantifiable**`, I'd write:
`xref:bamm-specification:ROOT:characteristics.adoc#quantifiable-characteristic[bamm-c:Quantifiable]`

Reply:
done